frameworks/Rust/viz/viz.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.72
+FROM rust:latest
```

Comment:
Please keep this locked down to a specific version

Reply:
Moved to #8658